build_caldera_api.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Before running: 
+# - go install github.com/go-swagger/go-swagger/cmd/swagger@latest
+
+set -e
+PROJ_PATH="$PWD"
+
+
+CALDERA_VERSION="e4a4b9a320e707e58e69e3213ca07fdc561350ce" # Commit hash
+OUTPUT_BASE_FOLDER="./internal/capability/caldera/api"
+
+OUTPUT="$PROJ_PATH/caldera_api_raw.json"
+
+DOCKER_CMD="podman"
+# DOCKER_CMD="docker"
```

Comment:
Either keep Docker the default or auto-discover it

Reply:
:(

Reply:
I mean Docker is the standard, just `alias docker podman` if you want to use podman 😂

Reply:
(or auto-discover, is Podman is there just use it)